cfgov/cfgov/settings/base.py

```diff
@@ -459,11 +457,8 @@
 if ENABLE_CLOUDFRONT_CACHE_PURGE:
     WAGTAILFRONTENDCACHE["files"] = {
         "BACKEND": "wagtail.contrib.frontend_cache.backends.CloudfrontBackend",
-        "DISTRIBUTION_ID": {
-            "files.consumerfinance.gov": os.environ.get(
-                "CLOUDFRONT_DISTRIBUTION_ID_FILES"
-            )
-        },
+        "DISTRIBUTION_ID": os.environ.get("CLOUDFRONT_DISTRIBUTION_ID_FILES"),
+        "HOSTNAMES": ["files.consumerfinance.gov", ],
```

Comment:
Any reason not to make this environment variable required? If we've defined `ENABLE_CLOUDFRONT_CACHE_PURGE` in the environment we should also be defining `CLOUDFRONT_DISTRIBUTION_ID_FILES`. If we haven't for some reason, and this gets set as `None`, then we won't see a failure until such time as the purge is attempted. Better to see the failure at settings import time.

```suggestion
        "DISTRIBUTION_ID": os.environ["CLOUDFRONT_DISTRIBUTION_ID_FILES"],
        "HOSTNAMES": ["files.consumerfinance.gov"],
```